### PR TITLE
Add Xiaomi 11T overlays

### PR DESCRIPTION
**Pull request by Mahmoud-SE**
Wasn't tested as of now, could be left on hold until it's confirmed.

### CODE REVIEW

File: Xiaomi/Mi11T/res/values/notch.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+** See the License for the specific language governing permissions and
+** limitations under the License.
+*/
+-->
+<resources>
+    <dimen name="rounded_corner_radius_top">0.0dip</dimen>
+    <dimen name="rounded_corner_radius_bottom">0.0dip</dimen>
+    <dimen name="status_bar_height">24.0dip</dimen>
+    <dimen name="status_bar_height_portrait">24.0dip</dimen>
+    <string name="config_mainBuiltInDisplayCutout" />
```
Comment (phhusson) [inline Xiaomi/Mi11T/res/values/notch.xml:24]: Googling for Xiaomi Mi 11T definitely say there is a display cutout...

File: Xiaomi/Mi11T/AndroidManifest.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+        package="me.phh.treble.overlay.xiaomi.mi11t"
+        android:versionCode="1"
+        android:versionName="1.0">
+        <overlay android:targetPackage="android"
+                android:requiredSystemPropertyName="ro.product.vendor.device"
+                android:requiredSystemPropertyValue="amber"
```
Comment (phhusson) [inline Xiaomi/Mi11T/AndroidManifest.xml]: I'm sorry, but this name is way too generic, it's highly likely other OEMs have/will give that name
You'll need to find another way to match the device

Comment (Mahmoud-SE) [reply Xiaomi/Mi11T/AndroidManifest.xml]: Alright, will do

**Comment by Mahmoud-SE** [comment]
Have to do another push. I'll close this temporarily

**Pull request closed without merging**